app/clien.py

```diff
@@ -0,0 +1,49 @@
+from typing import Type
+
+
+class Client:
```

Comment:
Use Customer name, don't change it, you have an example inside a task description